src/views/collectable/v3/Collectable.vue

```diff
@@ -188,7 +188,7 @@
             :winningAddress="winningAddress"
             @update-state="updateCollectableState"
           />
-          <div v-if="secondaryMarketListingsForRender?.length > 0" class="text-3xl font-title font-bold text-center mb-6 mt-12" :class="darkMode && 'dark-mode-text'">
```

Comment:
aligns the title of this block to the left - like we did for the other items in this list.